src/js/components/Layer/LayerContainer.js

```diff
@@ -243,11 +243,16 @@ const LayerContainer = forwardRef(
       );
     }
 
-    if (theme.layer.background) {
+    const themeContext = useMemo(() => {
```

Comment:
I'm wondering if we should be clearer in naming this to be `themContextValue` since it isn't really a ThemeContext.

Reply:
Addressed.